lib/dynflow/active_job/queue_adapter.rb

```diff
@@ -28,6 +28,10 @@ def plan(attributes)
         def run
           input[:job_class].constantize.perform_now(*input[:job_arguments])
         end
+
+        def label
+          input[:job_class]
```

Comment:
AFAIK you pass the args to the `perform` method of the active job: I don't see where in the active job class you would be able to define the label.

The label here is just do disctinguish differend kinds of jobs, but for your case, I believe there should be something on application layer to link the resources and jobs. We use foreman-tasks for it right now though locks. I believe we need a counterpart in foreman.